README.md

```diff
@@ -46,3 +46,28 @@ A version of [convergent encryption](http://en.wikipedia.org/wiki/Convergent_enc
 
 #Video of the process 
 [self_encryption process and use case video] (https://www.youtube.com/watch?v=Jnvwv4z17b4)
+
+#Basic self_encryptor example DIY instructions
+
+Instructions to use the 'basic_encryptor' example to encrypted a file into encrypted chunks and a datamap and then decrypted the datamap and restore the file.
```

Comment:
Cheers Nick I updated the PR with your feedback but omitted the "which is also split into chunks and encrypted" part as I don't believe the example does this part. Thanks :-)